Kitodo-DataFormat/src/main/java/org/kitodo/dataformat/access/FileXmlElementAccess.java

```diff
@@ -144,14 +154,20 @@ DivType toDiv(Map<URI, FileType> mediaFilesToIDFiles,
         div.setTYPE(physicalDivision.getType());
         for (Entry<MediaVariant, URI> use : physicalDivision.getMediaFiles().entrySet()) {
             Fptr fptr = new Fptr();
-            fptr.setFILEID(mediaFilesToIDFiles.get(use.getValue()));
+            Object fileId = mediaFilesToIDFiles.get(use.getValue());
+            if (PhysicalDivision.TYPE_TRACK.equals(physicalDivision.getType()) && MediaUtil.isAudioOrVideo(
+                    use.getKey().getMimeType()) && physicalDivision.hasMediaPartialView()) {
+                fptr.setArea(getAreaType(fileId));
+            } else {
+                fptr.setFILEID(fileId);
```

Comment:
Yes, because it is set as an area attribute.

```
            <mets:fptr>
                <mets:area FILEID="FILE_XXXX_DEFAULT" BEGIN="00:00:00" BETYPE="TIME" EXTENT="00:02:30" EXTTYPE="TIME"></mets:area>
            </mets:fptr>
```